app/controllers/cases/offender_sar_controller.rb

```diff
@@ -19,13 +19,6 @@ def new
       permitted_correspondence_types
       authorize case_type, :can_add_case?
 
-      @case = OffenderSARCaseForm.new(session)
-      step = params[:step].present? && params[:step] != 'new' ? params[:step] : @case.steps.first
-
-      # @todo: Why does current_step need to be set? is it something OffenderSARCaseForm
-      # can figure out itself?
-      @case.current_step = step
-
```

Comment:
Not related to this PR per se, but cleaned up some redundant code, partly left from merging work